packages/data-stores/src/reader/mutations/use-subscription-manager-user-settings-mutation.ts

```diff
@@ -14,7 +15,23 @@ const useSubscriptionManagerUserSettingsMutation = () => {
 				body: data,
 				isLoggedIn,
 			} );
+			if ( settings.errors ) {
+				if ( settings.errors.invalid_input ) {
+					throw new Error( settings.errors.invalid_input.join( ', ' ) );
+				} else {
+					throw new Error(
+						translate( 'Something went wrong.', {
+							context: 'Something went wrong will saving the user settings',
```

Comment:
This looks like a typo.

```suggestion
							context: 'Something went wrong with saving the user settings',
```

Reply:
Neve mind. This doesn't seem to be in the trunk anymore.

Reply:
It actually is in the current `trunk`. My VS Code was just loading old version of the file for some reason.

Reply:
Thanks for noticing! I'll try to fix this in a future PR.